### PR TITLE
YG-434: Added Print Modal and Print Now feature on member search page.

### PR DESCRIPTION
**Pull request by pepcus-akshita**
# Related issues
-  Added Print Modal and Print Now feature on member search page.

# Proposed changes
- Print Now option is enabled when any student is selected and it prints the Id card for selected students(s).

# Screenshots

image

image

image

### CODE REVIEW

File: src/v2/configs/tenant/YJSG-Indore/strings.json [hunk 1]
```diff
@@ -288,5 +288,7 @@
 	"ACTIVE": "Active",
 	"INACTIVE": "Inactive",
 	"SEE_ALL": "See All",
-	"SEE_LESS": "See Less"
+	"SEE_LESS": "See Less",
+	"COORDINATOR_NAME": "Coordinator name: ",
```
Comment (pepcus-mohit) [inline src/v2/configs/tenant/YJSG-Indore/strings.json:292]: Do not include symbols on strings.json.
```suggestion
	"COORDINATOR_NAME": "Coordinator name",
```

File: src/v2/layouts/MemberDataGrid/MemberDataGridContainer.jsx [hunk 2]
```diff
@@ -126,49 +127,54 @@ export const MemberDataGridContainer = () => {
...
-			/>
-			<StyledMemberConfig>
-				<MemberSearch
-					onSearch={onSearch}
-					studentList={() => {
-						setStudentList(fetchStudentResp?.studentList);
-					}}
+		<>
+			<StyledWrapper>
+				<Header showLogo={true}>
```
Comment (pepcus-mohit) [inline src/v2/layouts/MemberDataGrid/MemberDataGridContainer.jsx:132]: Can you coordinate with Shivani as she has made some changes in the Buttons on Admin Grid and that already includes a Logout button.
So I think we don't need the Logout button here. May be we can replace the Buttons Logout link implementation with this logout button component.

File: src/v2/utils/dataGridUtils.js [hunk 3]
```diff
@@ -217,3 +219,33 @@ export const getTwelveHoursFormatTime = (date) => {
 
 	return `${hours}:${minutes} ${ampm}`;
 };
+
+export const getFormattedMemberId = ({ memberId }) => {
```
Comment (pepcus-mohit) [inline src/v2/utils/dataGridUtils.js:223]: How about this ?
String(num).padStart(4,'0')

**Review by pepcus-mohit — CHANGES_REQUESTED** [review]

File: src/v2/layouts/MemberDataGrid/MemberDataGridContainer.jsx (hunk 2 above)
Comment (pepcus-akshita) [reply src/v2/layouts/MemberDataGrid/MemberDataGridContainer.jsx:132]: As per discussion, we don't need this logout component here. So removing this from here.

**Review by pepcus-mohit — APPROVED** [review]

**Pull request merged**